File: dayFour/Javascript/TransactionLogApp.js
const scanner = require("prompt-sync")();

let balance = 0;
let amount = 0;

let getBalance=()=>{
    return balance;
}

//console.log(getBalance());

let deposit=(newAmount)=>{
    amount = newAmount;
    newBalance = amount + balance;
    balance = newBalance;
}

let transactionHistory=()=>{
    let reply = "Deposited: #" + amount + " | New Balance: #" + balance;
    return reply;
}

//amount = 1000;
//
//deposit(amount);
//
//console.log();
//console.log(getBalance());
//console.log(transactionHistory());

File: dayFour/Java/With Test/TransactionLogApp.java
public class TransactionLogApp {

    public static double getAccountBalance(){
        double accountBalance = 0;
        
        return accountBalance;
    }
    
    public static double deposit(double amount, double balance){
        
        double newBalance = amount + balance;
        
        return newBalance;
    }
    
    public static String getTransactionHistory(double amount, double balance){
        String format = "Deposited: #" + amount + " | New Balance: #" + deposit(amount,balance);
        
        return format;
    }
}
